cmd/bedMaxWig/bedMaxWig_test.go

```diff
@@ -32,3 +32,27 @@ func TestBedMaxWig(t *testing.T) {
 		}
 	}
 }
+
+var bedMaxWigNormFlagTest = []struct {
```

Comment:
This is just a stylistic comment but I think bedMaxWigTests and bedMaxWigNormFlagTest can be merged, and they can both use TestBedMaxWig rather than having to use TestBedMaxWigNormFlag, if the bedMaxWigTests struct has 1 more field that is "norm bool", so you can distinguish between the true and false cases, and in the TestBedMaxWig for loop norm status can be specified with v.norm

Reply:
That's how I originally wrote this (with them merged) but had caught a snag. I cant remember what it was now though. I know after a conversation with Craig he said to do them separate, and I think that is generally how the test files are written? Where we would have a separate test for each flag? Maybe that makes it easier to troubleshoot problems/test fails?

Reply:
Interesting! That's good to know!

Reply:
I think either way could work.  I can't remember exactly, but at the time I think it was helpful to separate them out to confirm that the non-normalized version was still working, but the normalized version was failing.